pay-queue/src/pay_queue/templates/csv_failed_email.html

```diff
@@ -0,0 +1,6 @@
+The following CSV feedback failed:
```

Comment:
Shouldn't this just be generic? that way we can use it for CSV, EJV, TD17?

Reply:
I think EJV is pushing to queue for now
![image](https://github.com/bcgov/sbc-pay/assets/16507223/c60bd0e7-437a-43d7-80ca-066375efb76a)

Reply:
nvm, they will change eventually, we have other two tickets:
https://app.zenhub.com/workspaces/relationships-team-space-new-649992c7f903a02c9b33009f/issues/gh/bcgov/entity/19495

Reply:
it should be consistent.. same kind of idea right?